airflow/settings.py

```diff
@@ -30,7 +30,7 @@
 from sqlalchemy.orm.session import Session as SASession
 from sqlalchemy.pool import NullPool
 
-import airflow
+# noinspection PyUnresolvedReferences
```

Comment:
Why do we need this ?

Reply:
Same as before - Pycharm's "green checkmark". But if you think it's annoying I can remove those